kernel/kernel-impl/src/main/sql/oracle/sakai_config_item.sql

```diff
@@ -27,12 +27,4 @@ ALTER TABLE SAKAI_CONFIG_ITEM
 CREATE INDEX SCI_NODE_IDX ON SAKAI_CONFIG_ITEM (NODE ASC);
 CREATE INDEX SCI_NAME_IDX ON SAKAI_CONFIG_ITEM (NAME ASC);
 
-CREATE SEQUENCE SAKAI_CFG_ITEM_SEQ;
-
-CREATE OR REPLACE TRIGGER SCI_ID_AI
-BEFORE INSERT ON SAKAI_CONFIG_ITEM
-FOR EACH ROW
-BEGIN
-  SELECT SAKAI_CFG_ITEM_SEQ.NEXTVAL INTO :new.ID FROM dual;
-END;
-/
+CREATE SEQUENCE SAKAI_CONFIG_ITEM_S;
```

Comment:
What I mean is, the `before insert` trigger assigns the auto increment ID to every record before saving. If the sequence isn't being used manually in code somewhere else (which I doubt it is, since you renamed it), then this trigger has to stay, but be updated to use the newly named sequence. Have you run this on Oracle?